content/md/en/docs/tutorials/build-a-blockchain/upgrade-a-running-network.md

```diff
@@ -149,7 +149,7 @@ To update the dependencies for the runtime to include the Utility pallet:
    codec = { package = "parity-scale-codec", version = "3.0.0", default-features = false, features = ["derive"] }
    scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
    
-   pallet-aura = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
+   pallet-aura = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/polkadot-sdk.git", branch = "polkadot-v1.0.0" }
```

Comment:
:(

Reply:
The new repo should be available in 2 weeks, clarified this with Sacha. Will change it after the repo is available, and till then we can go ahead with this.